doc/source/io.rst

```diff
@@ -79,116 +79,140 @@ for some advanced strategies
+    behavior.
+  - ``skipinitialspace``: boolean, default ``False``. Skip spaces after
+    delimiter.
+  - ``escapechar``: str (length 1), default ``None``. One-character string
+    used to escape delimiter when quoting is ``QUOTE_NONE``.
+  - ``dtype``: Type name or dict of column -> type, default ``None``. Data type
+    for data or columns. E.g. ``{'a': np.float64, 'b': np.int32}`` (unsupported
+    with ``engine='python'``).
+  - ``compression``: {``'infer'``, ``'gzip'``, ``'bz2'``, ``None``}, default
+    ``'infer'``. For on-the-fly decompression of on-disk data. If 'infer', then
+    use gzip or bz2 if filepath_or_buffer is a string ending in '.gz' or '.bz2',
+    respectively, and no decompression otherwise. Set to ``None`` for no
+    decompression.
+  - ``dialect``: string or :class:`python:csv.Dialect` instance, default
+    ``None``. If ``None`` defaults to Excel dialect. Ignored if sep longer than
+    1 char. See :class:`python:csv.Dialect` documentation for more details.
+  - ``header``: int or list of ints, default ``'infer'``.  Row number(s) to use as
+    the column names, and the start of the data. Default behavior is as if
+    ``header=0`` if no ``names`` passed, otherwise as if ``header=None``.
+    Explicitly pass ``header=0`` to be able to replace existing names. The
+    header can be a list of integers that specify row locations for a
+    multi-index on the columns e.g. ``[0,1,3]``. Intervening rows that are not
+    specified will be skipped (e.g. 2 in this example is skipped). Note that
+    this parameter ignores commented lines and empty lines if
+    ``skip_blank_lines=True``, so header=0 denotes the first line of data
+    rather than the first line of the file.
+  - ``skiprows``: list-like or integer, default ``None``. Line numbers to skip
+    (0-indexed) or number of lines to skip (int) at the start of the file.
+  - ``index_col``:  int or sequence or ``False``, default ``None``. Column to
+    use as the row labels of the DataFrame. If a sequence is given, a MultiIndex
+    is used. If you have a malformed file with delimiters at the end of each
+    line, you might consider ``index_col=False`` to force pandas to *not* use
+    the first column as the index (row names).
+  - ``names``: array-like, default ``None``. List of column names to use. If
+    file contains no header row, then you should explicitly pass
+    ``header=None``.
+  - ``prefix``: str, default ``None``. Prefix to add to column numbers when
+    no header, e.g. 'X' for X0, X1, ...
+  - ``na_values``: str, list-like or dict, default ``None``. Additional strings
+    to recognize as NA/NaN. If dict passed, specific per-column NA values.
+  - ``true_values``: list, default ``None``. Values to consider as ``True``.
+  - ``false_values``: list, default ``None``. Values to consider as ``False``.
+  - ``keep_default_na``: bool, default ``True``. If na_values are specified and
+    keep_default_na is ``False`` the default NaN values are overridden,
```

Comment:
Fixed by inserting the actual values, although I'm not sure if the added complexity is worth it.